pkg/data/components/S3Exporter.yaml

```diff
@@ -53,14 +54,18 @@ properties:
     description: |
       The timeout to use for the S3. The default value is `5s`.
     type: duration
-    validations: [duration]
+    default: 5s
+    validations:
+      - duration
     advanced: true
   - name: Marshaler
     summary: The marshaler to use for the S3.
     description: |
       The marshaler to use for the S3. The default value is `otlp_json`.
     type: string
```

Comment:
any option that uses a oneof validation may benefit from a different type than string

Reply:
Sure, I'll open a issue to track adding discrete types when using `oneof` validations.